xds/src/main/java/io/grpc/xds/ClusterResolverLoadBalancer.java

```diff
@@ -127,6 +127,7 @@ public void handleResolvedAddresses(ResolvedAddresses resolvedAddresses) {
       this.config = config;
       delegate.handleResolvedAddresses(resolvedAddresses);
```

Comment:
We will do a second pass to replace this call with acceptResolvedAddresses and propagate the return value? That sounds fair to me.